### PR TITLE
Support Sprockets 3

### PR DESCRIPTION
**Pull request by camertron**
Adds support for sprockets 3. Not much needs to change, really.

### CODE REVIEW

**Comment by steel** [comment]
Thanks for the PR!

File: sprockets-derailleur.gemspec [hunk 1]
```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency 'sprockets', "~>2"
+  gem.add_dependency 'sprockets', '>= 2'
```
Comment (PikachuEXE) [inline sprockets-derailleur.gemspec]: This version constraint seems too open to me
I understand Sprockets 4 beta is out but we should not assume this gem will support sprockets 5 yet

Comment (camertron) [inline sprockets-derailleur.gemspec]: Yeah, I thought about that. The thing is, if the gem does by happenstance work with Sprockets 5, then we don't have any additional work to do. If it doesn't, then someone will report it and we can publish a fix. Schneems himself advocates this approach.

File: lib/sprockets-derailleur/manifest.rb [hunk 2]
```diff
@@ -113,15 +112,12 @@ def worker(paths)
                 if File.exist?(target)
                   logger.debug "Skipping #{target}, already exists"
                 else
-                  logger.debug "Writing #{target}"
+                  logger.info "Writing #{target}"
                   asset.write_to target
-                  asset.write_to "#{target}.gz" if asset.is_a?(BundledAsset)
```
Comment (PikachuEXE) [inline lib/sprockets-derailleur/manifest.rb]: `if asset.is_a?(BundledAsset)` means
generate gzip, not skipping
unless I got it wrong

Comment (daveallie) [reply lib/sprockets-derailleur/manifest.rb:118]: I just came across the same thing, opened a PR: https://github.com/steel/sprockets-derailleur/pull/28

**Pull request merged**